wsclient/src/test/java/org/hpccsystems/ws/client/FileSprayClientTest.java

```diff
@@ -187,10 +188,11 @@ public static DropZoneWrapper getDropzone(List<DropZoneWrapper> dropzones, Strin
         return null;
     }
 
+    @Category(IntegrationTest.class)
     @Test
     public void testUploadFile()
     {
-        File uploadFile = new File("src/test/resources/fileSprayTest/salesdata_orig.csv");
+        File uploadFile = new File("src/test/resources/fileSprayTest/Japi274deleteDropzoneFileTest.csv");
```

Comment:
Does this file actually exist?

Reply:
Yes. I re-named the file before submitting my last PR but didn't change the name here.